mmv1/third_party/terraform/tests/data_source_google_firebase_android_app_test.go

```diff
@@ -0,0 +1,52 @@
+package google
+
+import (
```

Comment:
This file needs similar tags to prevent it from appearing in GA where the resource doesn't exist

Reply:
Done.

Reply:
@slevenick Looks like the tags only work in .go.erb files.

Reply:
You can rename the file to .go.erb

Reply:
Done.